Marlin/src/module/planner.h

```diff
@@ -273,7 +280,7 @@ typedef struct block_t {
      * Using OCR instead of raw power, because it avoids
      * floating point operations during the move loop.
      */
-    uint8_t power;
+    volatile uint8_t power;
```

Comment:
We have to be careful with `volatile` vars outside of interrupts. It's a good idea to make a non-volatile copy at the start of logic and use the copy, because otherwise the value will change in the middle of your logic or math.

Reply:
Noted: It should be ok in this case as it always an output value and not read for any calculations.